packages/repository/src/repositories/relation.repository.ts

```diff
@@ -85,9 +88,11 @@ export class DefaultHasManyEntityCrudRepository<
     filter?: Filter | undefined,
     options?: Options,
   ): Promise<T> {
-    // throw new Error('Method not implemented.');
-    filter = constrainFilter(filter, this.constraint);
-    return await this.targetRepository.findById(id, filter);
+    throw new Error(
```

Comment:
-1 for runtime checks. We should leverage type system to reject invalid repository usage at compile time.

Reply:
>  the method is not available in the HasManyEntityCrudRepository interface?

exactly 👍